browse-scaler/src/cmr.js

```diff
@@ -1,9 +1,10 @@
 const fetch = require('node-fetch');
 
 /* CMR ENVIRONMENT VARIABLES */
-const cmrRootUrl = `http://${process.env.CMR_ROOT}`;
+const cmrRootUrl = `https://${process.env.CMR_ROOT}`;
```

Comment:
`CMR_ROOT` points to the services load balancer? I don't think the lb can do https.

Reply:
Ah it does - I may want to rethink how I'm doing this then